specification/search/data-plane/Azure.Search/preview/2024-05-01-Preview/searchindex.json

```diff
@@ -0,0 +1,3097 @@
+          }
+        ]
+      },
+      "description": "The way the field was used for the semantic enrichment process."
+    },
+    "QueryResultDocumentRerankerInput": {
+      "type": "object",
+      "properties": {
+        "title": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The raw string for the title field that was used for semantic enrichment."
+        },
+        "content": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The raw concatenated strings for the content fields that were used for semantic enrichment."
+        },
+        "keywords": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The raw concatenated strings for the keyword fields that were used for semantic enrichment."
+        }
+      },
+      "description": "The raw concatenated strings that were sent to the semantic enrichment process."
+    },
+    "VectorsDebugInfo": {
+      "type": "object",
+      "properties": {
+        "subscores": {
+          "$ref": "#/definitions/QueryResultDocumentSubscores",
+          "readOnly": true,
+          "description": "The breakdown of subscores of the document prior to the chosen result set fusion/combination method such as RRF."
+        }
+      }
+    },
+    "QueryResultDocumentSubscores": {
+      "type": "object",
+      "properties": {
+        "text": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/TextResult"
+          },
+          "readOnly": true,
+          "x-nullable": true,
+          "description": "The BM25 or Classic score for the text portion of the query."
+        },
+        "vectors": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/QueryResultDocumentVectorSubscores"
+          },
+          "readOnly": true,
+          "x-nullable": true,
+          "description": "The vector similarity and @search.score values for each vector query."
+        }
+      },
+      "description": "The breakdown of subscores between the text and vector query components of the search query for this document. Each vector query is shown as a separate object in the same order they were received."
+    },
+    "TextResult": {
+      "type": "object",
+      "properties": {
+        "searchScore": {
+          "type": "number",
+          "format": "double",
+          "readOnly": true,
+          "description": "The BM25 or Classic score for the text portion of the query."
+        }
+      },
+      "readOnly": true,
+      "additionalProperties": true,
+      "description": "The BM25 or Classic score for the text portion of the query."
+    },
+    "QueryResultDocumentVectorSubscores": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "object",
+        "items": {
+          "$ref": "#/definitions/SingleVectorFieldResult"
+        },
+        "readOnly": true
+      },
```

Comment:
A schema with `type: object` should not specify `items` -- `items` only applies for `type: array`.

Reply:
it should actually be an array (since we are going to have one entry per vector-field pair)